antismash/detection/hmm_detection/cluster_rules/strict.txt

```diff
@@ -503,8 +503,8 @@ RULE phosphonate
     CATEGORY other
     DESCRIPTION Phosphonate
     CUTOFF 20
-    NEIGHBOURHOOD 20
-    CONDITIONS phosphonates
+    NEIGHBOURHOOD 5
+    CONDITIONS (phosphonatesV2 and (2-Hacid_dh_C or Aldedh or Aminotran_1_2 or Aminotran_5 or AurF or CTP_transf_like or DUF4992 or Fe-ADH or HMGL-like or Metallophos_2 or NTP_transf_3 or NTP_transf_5 or PALP or PEP_mutase or TPP_enzyme_C or TPP_enzyme_N or phosphonatesV2) and not phosphonatesHydrolase and not phosphonatesMeIsocitrateLyase)
```

Comment:
The second section of this condition will always be satisfied if the first section is, because it's including the first condition as an `or` (i.e. `A and (A or ...)` makes all of the other included 'or' terms irrelevant. Is this intentional?

Also, it would probably be better to move the old `phosphonates` profile to a new name and have just `phosphonates` instead of the `V2` version.

Reply:
Renamed the new rule. Repeating the phosphonates model was intentional, the additional models are not required but allow tighter neighbourhood value for most clusters (i.e. a cluster with only the phosphonate hit will still be detected)